src/shared/domain/definition/case-field.model.ts

```diff
@@ -1,7 +1,7 @@
 import { Orderable } from '../order';
```

Comment:
I think we must add some good unit tests for this class. With some examples of the json of a dynamic list

Reply:
added